argument_parser.py

```diff
@@ -60,5 +72,14 @@ def get_parameters():
                         The correct format should be YYYY-MM-DD""",
                         default=current_date)
 
+    parser.add_argument('-temp', metavar='temperature', type=str, choices=['C', 'F'],
```

Comment:
By the FD, this one is named `-temperature`, not `-temp`. When I'm trying to execute with the extended name:
```
> python .\main.py -location "New York" -date 2024-06-17 -temperature "F" -weather_components "pressure"
2024-06-16
usage: BetterWeather [-h] [-location location] [-fp forecast_period] [-date date] [-temp temperature] [-weather_components weather_components]
> python .\main.py -location "New York" -date 2024-06-17 -temperature "C" -weather_components "pressure"
2024-06-16
usage: BetterWeather [-h] [-location location] [-fp forecast_period] [-date date] [-temp temperature] [-weather_components weather_components]
BetterWeather: error: unrecognized arguments: -temperature C
```
Also, for the `choices`, why we don't reuse the already existing variable here?

Reply:
@KrzysiekJodlowski Solved.